tools/install_prerequisites.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -eu
+
+# Set the CWD to Omicron's source.
+SOURCE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+cd "${SOURCE_DIR}/.."
+
+HOST_OS=$(uname -s)
+if [[ "${HOST_OS}" == "Linux" ]]; then
+  echo "Linux"
+  sudo apt-get install libpq-dev
+  sudo apt-get install pkg-config
+elif [[ "${HOST_OS}" == "SunOS" ]]; then
+  echo "illumos"
+  need=(
+    'build-essential'
+    'library/postgresql-13'
+    'pkg-config'
+    'brand/omicron1/tools'
+    'pkg:/package/pkg'
```

Comment:
Should this happen first, and maybe unconditionally?

Reply:
Sure, I can make it the first install.

I've also just updated the tooling here to make the call to `pkg install` unconditionally, which should act as both "update" and "install" - I'll just check the error code "no updates to install (4)", and explicitly allow it.

Reply:
Source on the usage of "4", since it's a little magic:

https://man.omnios.org/man1/pkg.1#:~:text=No%20changes%20were%20made%20%2D%20nothing%20to%20do.